src/scss/commons/components.scss

```diff
@@ -35,4 +37,18 @@
   height: 30px;
   outline: 0;
   padding: 5px;
-}
\ No newline at end of file
+}
+
+.error-label {
+  color: $red;
+  font-size: 14px;
+  margin: 5px auto;
+}
+
+.error-input {
+  border: 1px solid $red;
+}
+
+.valid-input {
```

Comment:
I think this is not part of the design and adds a lot of overhead to the class logic

Reply:
Ok, I'll remove it! :ok_hand: